docs/showcase/modules/ROOT/pages/push-notifications.adoc

```diff
@@ -0,0 +1,8 @@
+
+
+= {unifiedpush-service}
+
+
+. Send a push notification as described in
+link:{push-guide-link}#sending[Sending a Push Notification]
```

Comment:
What we do with this? I think that push should link to showcase same with sync. Not the other way around